examples/moving_least_squares/moving_least_squares.cpp

```diff
@@ -20,7 +20,8 @@ using Point = ArborX::Point<2, double>;
 
 KOKKOS_FUNCTION double functionToApproximate(Point const &p)
 {
-  return Kokkos::cos(p[0] + p[1] / 4);
+  (void)p;
+  return 10 * p[1]; // Kokkos::cos(p[0] + p[1] / 4);
```

Comment:
Do we need any changes in this example file? It seems that we better have unit tests for specific cases.